Float.Core/UX/NavigationPageNavigationContext.cs

```diff
@@ -104,6 +105,45 @@ public void DismissPage(bool animated = true)
             });
         }
 
+        /// <inheritdoc />
+        public async Task DismissPageAsync(bool animated = true)
+        {
+            await DeviceProxy.InvokeOnMainThreadAsync(async () =>
+            {
+                if (HasModal)
+                {
+                    await navigationPage.Navigation.PopModalAsync(animated);
+                }
+            });
+        }
+
+        /// <inheritdoc/>
+        public async Task PushPageAsync(Page page, bool animated = true)
+        {
+            await DeviceProxy.InvokeOnMainThreadAsync(async () =>
+            {
+                await navigationPage.PushAsync(page, animated);
+            });
+        }
+
+        /// <inheritdoc/>
+        public async Task PopPageAsync(bool animated = true)
+        {
+            await DeviceProxy.InvokeOnMainThreadAsync(async () =>
+            {
+                await navigationPage.PopAsync(animated);
+            });
+        }
+
+        /// <inheritdoc/>
```

Comment:
@EBusch I think all of these should be consistent. I see `/// <inheritdoc/>` and `/// <inheritdoc />`

Reply:
@bbeversdorf yeah I think you're right. Annoyingly, Visual Studio has updated the default behavior of fixing these. I will make them consistent.